devfile.yaml

```diff
@@ -76,33 +76,23 @@ commands:
         isDefault: true
 
   - id: run-with-hsqldb
-    exec:
-      component: tools
-      workingDir: ${PROJECT_SOURCE}
-      commandLine: >-
-        java -jar target/*.jar
-      group:
-        kind: run
-        isDefault: true
-
-  - id: run-with-mysql
     exec:
       component: tools
       workingDir: ${PROJECT_SOURCE}
       commandLine: |
-        java -jar -Dspring.profiles.active=mysql \
-        -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5005 \
-        target/*.jar
+        mvn spring-boot:run -DskipTests \
+        -Dspring-boot.run.jvmArguments='-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005,quiet=y'
       group:
         kind: run
         isDefault: true
 
-  - id: run-debug
+  - id: run-with-mysql-new
```

Comment:
i think `run-with-mysql` would be better

Reply:
Yes, it's definitely better to use command name without `new` ending.
Thanks.